### DISCUSSION TITLE
Where are the debugging mono.dll files from the wiki?

### DISCUSSION THREAD

**Discussion by alex91ar** [dnSpyEx/dnSpy#118] [Q&A]
Hi all!
I'm trying to debug a game made on Unity just for fun, I looked at the [wiki link here](https://github.com/dnSpyEx/dnSpy/wiki/Debugging-Unity-Games#debugging-unity-games) and it says that the debugging mono.dll files for each version are on the release section. 

The most I could find are the source for each mono.dll file ready for debugging, however I'd like to avoid having to compile them if possible. Where they removed for a reason? Am I missing something?

Thanks!

**Comment by BL1NDX3N0N** [comment]
The compiled binaries were removed and was possibly done in favor of not having to consistently maintain them for different releases of Unity or it did not fit the release structure of this project. Here was the original link:

- https://github.com/dnSpyEx/dnSpy/releases/unity

I also recommend you check this repository:

- https://github.com/dnSpyEx/dnSpy-Unity-mono

**Comment by ElektroKill** [comment]
Hi, the link originates from the original dnSpy wiki page. That release link used to point to a valid release at some point. However, around the time dnSpy got archived the unity release together with other older releases got deleted. When the dnSpyEx fork was created, there was no sign of this release, so I wasn’t able to copy it over to here. While I’d love to be able to provide compiled binaries, I currently don’t have the facilities to be able to compile all the patched mono versions.

There are some users who have managed to back up this release but I have no yet verified whether the files are safe to use or not. Classify the following links as use at your own risk and potentially unsafe:
* https://github.com/liesauer/Unity-debugging-dlls/releases/tag/v2020.07.10
* https://github.com/wh0am15533/Patched-Unity-Mono
